language/diem-framework/compiled/src/transaction_scripts.rs

```diff
@@ -16,11 +16,11 @@ use std::{convert::TryFrom, fmt, path::PathBuf};
 // This includes the script ABIs as binaries. We must use this hack to work around
 // a problem with Docker, which does not copy over the Move source files that would be be used to
 // produce these binaries at runtime.
-const TXN_SCRIPTS_ABI_DIR: Dir = include_dir!("transaction_scripts/abi");
+const TXN_SCRIPTS_ABI_DIR: Dir = include_dir!("legacy_transaction_scripts/abi");
```

Comment:
I wonder if it would be better to cordon off the legacy scripts into their own `legacy` module in order to further separate them from non-legacy things.

Reply:
Don't think that should be an issue, and think this is probably a good idea. I'll take a look at updating.